docs/pages/jpscore/jpscore_demos.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Demos
+keywords: simulation
+tags: [jpscore, test]
+sidebar: jupedsim_sidebar
+folder: jpscore
+permalink: jpscore_demos.html
+summary: Introduction of demos in jpscore.
+
+last_updated: Feb 04, 2019
+---
+
+## 01_corridor
+### Description
```

Comment:
Please change structure of the file, for example as follows:
## 01_corridor 

**Description**

**Settings**

Reply:
> Please change structure of the file, for example as follows:

Thanks, done